backend/routes/comments.js

```diff
@@ -0,0 +1,58 @@
+import express from "express";
+import { Comment } from "../models/comment.js";
+import { Stock } from "../models/stock.js";
+import { User } from "../models/user.js";
+
+const router = express.Router();
+
+router.get("/comments/:ticker", async (req, res) => {
```

Comment:
Curious, any reason to do /comments/:ticker instead of the other way like :ticker/comments?

Reply:
It looks better to me the first way but the other way makes sense.